agent/api/task/task_test.go

```diff
@@ -424,6 +406,14 @@ var (
 	defaultSCProtocol                      = "/tcp"
 )
 
+func getDefaultDynamicHostPortRange() (start int, end int) {
```

Comment:
This function `getDefaultDynamicHostPortRange()` has been reviewed in https://github.com/aws/amazon-ecs-agent/pull/3585.